crates/bevy_pbr/src/material.rs

```diff
@@ -180,6 +180,13 @@ pub trait Material: Asset + AsBindGroup + Clone + Sized {
     ) -> Result<(), SpecializedMeshPipelineError> {
         Ok(())
     }
+
+    /// Specify a batch key to use for shadows. Returning Some(value) allows batching of distinct materials which will produce
```

Comment:
Wording isn't very clear. It took me some time to realize that this would basically force-batch different materials together by grouping them by this arbitrary value, and then using the first material of the batch for all meshes in the batch.